src/Engine/Rendering/Renderer.hpp

```diff
@@ -338,6 +338,9 @@ class RA_ENGINE_API Renderer
      */
     virtual std::unique_ptr<uchar[]> grabFrame( size_t& w, size_t& h ) const;
 
+    /// return the value of the depth buffer under pixel (x,y)
+    ///
```

Comment:
It should be of importance to be more clear in which space the depth value is expected : screen space, NDC, camera space ?